assets/css/default.css

```diff
@@ -6,9 +6,9 @@ body { 	background-color: #ddd; color: #333;
 		font-family: "Verdana";
 }
 
-#header {}
-#mainmenu {}
-#contents {}
+#header { width: 100%; }
```

Comment:
complete your css before committing your files

Reply:
noted. this is just some tests